### PR TITLE
5512 to test

### PR DESCRIPTION
**Pull request by TomElliottFlexion**

### CODE REVIEW

File: .github/workflows/typescript-untouched-files-check.yml [hunk 1]
```diff
@@ -21,7 +21,7 @@ jobs:
         uses: actions/checkout@v4
         with:
           repository: ustaxcourt/ef-cms
-          ref: ${{ env.TARGET_BRANCH }}
+          ref: staging
```
Comment (nechama-krigsman) [inline .github/workflows/typescript-untouched-files-check.yml]: @cruzjone-flexion is this temp? or do want to merge this?

Comment (cruzjone-flexion) [reply .github/workflows/typescript-untouched-files-check.yml]: Temp

File: shared/src/business/useCases/generatePetitionPdfInteractor.ts [hunk 2]
```diff
@@ -0,0 +1,82 @@
+import {
+  CASE_TYPE_DESCRIPTIONS_WITHOUT_IRS_NOTICE,
+  CASE_TYPE_DESCRIPTIONS_WITH_IRS_NOTICE,
+} from '@shared/business/entities/EntityConstants';
+import {
+  ROLE_PERMISSIONS,
+  isAuthorized,
+} from '@shared/authorization/authorizationClientService';
+import { UnauthorizedError } from '@web-api/errors/errors';
+// import {
```
Comment (nechama-krigsman) [inline shared/src/business/useCases/generatePetitionPdfInteractor.ts]: can we remove?

Comment (TomElliottFlexion) [reply shared/src/business/useCases/generatePetitionPdfInteractor.ts]: affirmative

File: shared/src/business/entities/EntityConstants.ts [hunk 3]
```diff
@@ -1002,6 +1005,7 @@ export const CASE_TYPE_DESCRIPTIONS_WITH_IRS_NOTICE = {
 
 export const CASE_TYPE_DESCRIPTIONS_WITHOUT_IRS_NOTICE = {
   [CASE_TYPES_MAP.cdp]: 'CDP (Lien/Levy)',
+  // [CASE_TYPES_MAP.passport]: 'Passport',
```
Comment (nechama-krigsman) [inline shared/src/business/entities/EntityConstants.ts]: Can this be removed?

**Review by nechama-krigsman — APPROVED** [review]

File: shared/src/business/entities/EntityConstants.ts (hunk 3 above)
Comment (TomElliottFlexion) [reply shared/src/business/entities/EntityConstants.ts]: Good catch, that should not be commented out

**Pull request merged**